packages/lib/gpc/src/gpcCompile.ts

```diff
@@ -320,8 +321,12 @@ export function compileProofConfig(
   );
 
   // Create bounds check inputs
-  // TODO(POD-P2): Replace this with actual inputs.
-  const circuitBoundsCheckInputs = dummyBoundsCheckInputs(circuitDesc);
+  const boundsCheckConfig = boundsCheckConfigFromProofConfig(proofConfig);
```

Comment:
Since boundsCheckConfig isn't used after the call below, this line can be inside of compileProofBoundsChecks